### DISCUSSION TITLE
Second pass can't use prompt and negative prompt

### DISCUSSION THREAD

**Discussion by ChiaYen-Kan** [vladmandic/sdnext#2202] [General]
in new version(d833b853ffc3111e9fa6bb3408dcffad39aebeda) txt2img is can't use prompt and negative prompt in second pass, but in old version can do it 

why in new version can't use prompt and negative prompt in second pass?

**Comment by vladmandic** [comment]
which backend? which prompt are you refering to? take a minute to write a bit more details, otherwise i really don't know what you're talking about.

**Reply by ChiaYen-Kan** [reply]
in old version has  prompt and negative prompt area
![old](https://github.com/vladmandic/automatic/assets/40080620/72e5e2f0-4625-446a-a898-9baf42bb9ab6)

in new version is nothing
![new](https://github.com/vladmandic/automatic/assets/40080620/fdf0dfd5-2619-4cb3-99ee-ac8e4530f556)

backend is origin, i think this not a bug

**Comment by vladmandic** [comment]
that section is only valid for backend=diffusers, so in the new version its hidden if backend=original.
that prompt was NOT used for backend=original ever. it is used for backend=diffusers.

**Reply by ChiaYen-Kan** [reply]
so in old version with original backend it's just dummy operate, actual is no any effect in second pass?

PS. in old version the prompt of second pass be saved to image's metadata

**Reply by vladmandic** [reply]
> so in old version with original backend it's just dummy operate, actual is no any effect in second pass?

correct. it would be a valid enhancement to actually use it, but so far i've only implemented that for diffusers

> PS. in old version the prompt of second pass be saved to image's metadata

thats a mistake. that's why prompt is not shown anymore right now.
